lib/json/json.go

```diff
@@ -81,12 +82,8 @@ var Module = &starlarkstruct.Module{
 	},
 }
 
-func encode(thread *starlark.Thread, b *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	var x starlark.Value
-	if err := starlark.UnpackPositionalArgs(b.Name(), args, kwargs, 1, &x); err != nil {
-		return nil, err
-	}
-
+// Encodes the given Starlark value as a string in JSON format.
+func EncodeJSON(x starlark.Value) (string, error) {
```

Comment:
Let's call this Encode (since the package name is json) and make the result type a []byte.